### ISSUE TITLE
Minor error in URP when experimenting.

### ISSUE THREAD

**Issue by cdibbs** [crest-ocean/crest#600]
Hello! In the 4.4.0 URP version of OceanRenderer.cs, it would be handy if line 650-653 could read:

```c#
if (! System.Version.TryParse(pi.version, out var v) || v < requiredVersion)
{
   Debug.LogWarning($"Unsupported URP version {pi.version}. Crest requires a stable URP version " + requiredVersion + " or later, please update.", this);
}
```

That switches `Parse` to `TryParse`. Right now, it errors on 9.0.0-preview.35 for instance. Thanks!

**Comment by huwb** [comment]
Thanks, done, will be resolved on 4.5

**Comment by daleeidd** [comment]
4.5 is released which solves this issue. Please comment if you continue to experience problems.
